src/test/java/org/opensearch/performanceanalyzer/collectors/telemetry/RTFDisksCollectorTests.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.collectors.telemetry;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.performanceanalyzer.OpenSearchResources;
+import org.opensearch.performanceanalyzer.commons.metrics.AllMetrics;
+import org.opensearch.performanceanalyzer.commons.metrics.MetricsConfiguration;
+import org.opensearch.telemetry.metrics.Histogram;
+import org.opensearch.telemetry.metrics.MetricsRegistry;
+
+public class RTFDisksCollectorTests {
+    private RTFDisksCollector rtfDisksCollector;
+    private static MetricsRegistry metricsRegistry;
+    private static Histogram diskWaitTimeHistogram;
+    private static Histogram diskServiceRateHistogram;
+    private static Histogram diskUtilizationHistogram;
+
+    private static final String OS_TYPE = System.getProperty("os.name");
+
+    @Before
+    public void init() {
+        initMocks(this);
+        System.setProperty("performanceanalyzer.metrics.log.enabled", "False");
+        MetricsConfiguration.CONFIG_MAP.put(RTFDisksCollector.class, MetricsConfiguration.cdefault);
+
+        metricsRegistry = mock(MetricsRegistry.class);
+        diskWaitTimeHistogram = mock(Histogram.class);
+        diskServiceRateHistogram = mock(Histogram.class);
+        diskUtilizationHistogram = mock(Histogram.class);
+        OpenSearchResources.INSTANCE.setMetricsRegistry(metricsRegistry);
+
+        when(metricsRegistry.createHistogram(anyString(), anyString(), anyString()))
```

Comment:
Can we make it more predictable and simply use thenReturn to validate for all type of histograms. Here `diskUtilizationHistogram` will only be returned always.